index.js

```diff
@@ -279,6 +279,7 @@
   //  Accessible :: TypeClass
   var Accessible = Z.TypeClass(
     'sanctuary/Accessible',
+    readmeUrl('accessible-pseudotype'),
     [],
     function(x) { return x != null; }
   );
```

Comment:
We should define this type in sanctuary-def. The term _pseudotype_ is archaic, and I would rather link to the documentation of a sanctuary-def `Type` value than to a paragraph which makes Accessible appear special in some way.